src/services/itemLogin/index.ts

```diff
@@ -96,7 +97,21 @@ const plugin: FastifyPluginAsync = async (fastify) => {
       const member = notUndefined(user?.account);
       assertIsMember(member);
       return db.transaction(async (manager) => {
```

Comment:
I know it's not you, but I'm asking here as a global question. Because we are returning `db.transaction` without awaiting, what would happening if the code crash inside the transaction? Will the error handler catch the exception or will the error cause a server crash?